src/artemis/ispyb/store_in_ispyb.py

```diff
@@ -38,6 +42,11 @@ def store_grid_scan(self, full_params: FullParameters):
         self.run_number = self.detector_params.run_number
         self.omega_start = self.detector_params.omega_start
         self.xtal_snapshots = self.ispyb_params.xtal_snapshots_omega_start
+        self.upper_left = Point2D(
+            self.ispyb_params.upper_left.x, self.ispyb_params.upper_left.y
+        )
+        self.y_steps = full_params.grid_scan_params.y_steps
+        self.y_step_size = full_params.grid_scan_params.y_step_size
```

Comment:
**must**: You're storing these in `self` but when you actually write them in `_store_grid_info_table` you use the `full_parameters`

Reply:
Can you write a test that would have caught this too please?